native/xqlitenif/Cargo.toml

```diff
@@ -10,8 +10,11 @@ path = "src/lib.rs"
 crate-type = ["cdylib"]
 
 [dependencies]
-rusqlite = "0.32"
-rustler = { version = "0.36", features = ["nif_version_2_17"] }
+dashmap = { version = "6.1.0", default-features = false }
+r2d2 = "0.8.10"
+r2d2_sqlite = "0.27.0"
+rusqlite = "0.34"
+rustler = { version = "0.36.1", features = ["nif_version_2_17"] }
```

Comment:
I strongly doubt you need NIF version 2.17. We have no surface-level feature gated behind that. Even 2.16 is only required for UTF8 atoms, which are not implemented, yet. Just drop the feature reference entirely.

Reply:
Thank you. Done: https://github.com/dimitarvp/xqlite/pull/9/commits/d0cc00f725c64137f049067f74cf168b23d96ce4